### DISCUSSION TITLE
HDR appears too dark and leds produce incorrect colors in HDR mode

### DISCUSSION THREAD

**Discussion by inferno1231** [awawa-dev/HyperHDR#547] [Q&A]
Hi all,

I pulled a trigger recently and bought new AVStar splitter with HDMI 2.1, VRR etc. support and Elgato HD60X. Installed today (yay, finally working VRR with HyperHDR instance), it works mostly fine, but colours seem off. Right now color accuracy is much better when SDR to HDR in HyperHDR is disabled. Dark scenes appear too dark, white color becomes red in capture feed, etc. 

I have updated my HD60X to latest firmware, changed saturation/brightness settings to all 0 instead of 1/128/128 (I had MS2109 earlier), rebooted my Pi several times, uploaded correct LUT table that I downloaded from HyperHDR blog.
Logs look fine too - here is the link - https://pastebin.com/aG3HHihq

I just can't understand where lies the problem. Tried switching between NV12 and YUV codecs - no difference.

Any thoughts? 

It almost feels like HD60x is somehow capturing HDR stream by itself and HyperHDR LUT table is being applied onto HDR stream. Could it be?

**Comment by awawa-dev** [comment]
Hi
I'm not sure what causes this effect. I recommend performing an automatic LUT calibration for your system. For testing, you can also try connecting the video source directly to the elgato and the grabber itself to the TV to eliminate the splitter as a possible cause.

**Reply by inferno1231** [reply]
Sorry for the late answer, happy Easter!
I did LUT calibration, seems like it helped. Does it look ok?
https://pastebin.com/ZX2PAA80

To slightly another topic, HD60x started acting weirdly couple of days ago, exclusively in some PS5 games with VRR enabled. It started losing signal every couple of minutes. Have you ever experienced something like this?

But still, I am pleased with my current setup, as it allows me to fully use HDMI 2.1 potential of PS5 and Denon AVR + HyperHDR Ambilight. Finally :)

**Reply by awawa-dev** [reply]
Something strange is happening with the brown color and the overall error is very high. I can't say what could be causing this, most likely some colorspace settings on the source (e.g. forced by the EDID settings of some device in the HDMI chain). You can compare this with my results, which you will find in every package with a calibrated LUT.  
Regarding the occasional signal loss my HD60X has hundreds if not over a thousand hours of use and I've never noticed anything like this. If you are sure that the device to which the HD60x is connected provides sufficient current and there are no voltage spikes associated with power overload I would try to eliminate the HDMI splitter from the equation first, I think it may be responsible for this as well as some of your HDMI cables. I would leave the fault of the HD60X or its firmware for last.
